cmd/talosctl/cmd/docs.go

```diff
@@ -135,6 +136,10 @@ var docsCmd = &cobra.Command{
 					name:    "block",
 					fileDoc: block.GetFileDoc(),
 				},
+				{
+					name:    "pci",
```

Comment:
let's name it in some more generic way, e.g. `hardware`, otherwise we'll end up with tons of config folder